warehouse/query-core/src/main/java/datawave/query/transformer/GroupingTransform.java

```diff
@@ -153,6 +200,76 @@ public boolean hasNext() {
         return null;
     }
     
+    // @formatter:off
```

Comment:
Note that this only keeps the javadoc source from wrapping. The generated javadoc will still be wrapped, so you likely want to add the appropriate HTML markers to it to format it in a way that will be readable.